pkg/infrastructure/aws/clusterapi/aws.go

```diff
@@ -158,6 +160,75 @@ func (*Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
 	return nil
 }
 
+// Ignition info goes here
+func (p Provider) Ignition(ctx context.Context, in clusterapi.IgnitionInput) ([]*corev1.Secret, error) {
+	awsCluster := &capa.AWSCluster{}
+	key := k8sClient.ObjectKey{
+		Name:      in.InfraID,
+		Namespace: capiutils.Namespace,
+	}
+	if err := in.Client.Get(ctx, key, awsCluster); err != nil {
+		return nil, fmt.Errorf("failed to get AWSCluster: %w", err)
+	}
+
+	awsSession, err := in.InstallConfig.AWS.Session(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get aws session: %w", err)
+	}
+
+	lbNames := []string{awsCluster.Status.Network.APIServerELB.Name}
+	if awsCluster.Status.Network.SecondaryAPIServerELB.Name != "" {
+		lbNames = append(lbNames, awsCluster.Status.Network.SecondaryAPIServerELB.Name)
+	}
+
+	lbInput := elbv2.DescribeLoadBalancersInput{Names: aws.StringSlice(lbNames)}
+	res, err := elbv2.New(awsSession).DescribeLoadBalancersWithContext(ctx, &lbInput)
+	if err != nil {
+		return nil, fmt.Errorf("failed to descsribe load balancers: %w", err)
+	}
+
+	arns := []*string{}
+	for _, lb := range res.LoadBalancers {
+		splitArn := strings.Split(*lb.LoadBalancerArn, "/")
+		if len(splitArn) < 3 {
+			logrus.Debugf("unable to filter load balancer data with %s", *lb.LoadBalancerArn)
+			continue
+		}
```

Comment:
For the final implementation, let's use the SDK to parse the ARN. Here is an example from the destroy code: https://github.com/openshift/installer/blob/master/pkg/destroy/aws/aws.go#L393